lib/blacklight/solr/response/group_response.rb

```diff
@@ -32,6 +32,14 @@ def start
     params[:start].to_s.to_i
   end
 
+  ##
+  # Relying on an included super method fails here as it will not evaluate the
```

Comment:
Yeah that wasn't clear at all was it! I've updated the comment, let me know if there is better wording you prefer